stackdriver-core/docs/toc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "guides": [
+    {
+      "title": "FAQ",
+      "id": "faq",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/faq/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/faq/readme.md"
+    },
+    {
+      "title": "Troubleshooting",
+      "id": "troubleshooting",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/troubleshooting/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/troubleshooting/readme.md"
+    },
+    {
+      "title": "Contributing",
+      "id": "contributing",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/contributing/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/contributing/readme.md"
+    }
+  ],
+  "services": [
+    {
+      "title": "Stackdriver Core",
+      "type": "stackdriver/core"
+    }
```

Comment:
I don't think this PR is blocking beta. Currently there are no stackdriver packages using this new stackdriver-core gem yet. So we hold on merging this PR and refactoring logging to use this new stackdriver-core gem till after beta releases.